experimental/terraform/data_parser/outputs.tf

```
@@ -0,0 +1,3 @@
+output "cloud_run_endpoint" {
+    value = "TODO____MAKE_ME_HAPPEN"
```

📎 Did you mean to output the status url?
```suggestion
    value = "google_cloud_run_service.parser_service.status[0].url"
```

---

Er, yes... I suppose I did mean to. Fixed!